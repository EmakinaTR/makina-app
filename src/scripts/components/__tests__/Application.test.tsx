import * as React from 'react'
import { mount } from 'enzyme'
import Application from '../Application'

describe('<Application />', () => {
  it('renders successfully', () => {
    const component = mount(<Application />)
    expect(component).toBeTruthy()
  })
})
