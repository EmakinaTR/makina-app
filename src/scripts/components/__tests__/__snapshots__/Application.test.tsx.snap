// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Application /> renders successfully 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Application />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": "Hello World",
      "color": "primary",
      "variant": "contained",
    },
    "ref": null,
    "rendered": "Hello World",
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "Hello World",
        "color": "primary",
        "variant": "contained",
      },
      "ref": null,
      "rendered": "Hello World",
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
